tests/NuGet.Services.AzureSearch.FunctionalTests/Relevancy/V3RelevancyFunctionalTests.cs

```diff
@@ -42,8 +42,8 @@ public static IEnumerable<object[]> EnsureFirstResultsData()
 
             yield return new object[] { "tags:\"aws-sdk-v3\"", new[] { "awssdk.core", "awssdk.s3" } };
 
-            yield return new object[] { "entityframework", new[] { "entityframework" } };
-            yield return new object[] { "entity framework", new[] { "entityframework" } };
+            yield return new object[] { "entityframework", new[] { "entityframework", "microsoft.entityframeworkcore" } };
+            yield return new object[] { "entity framework", new[] { "microsoft.entityframeworkcore", "microsoft.entityframeworkcore.relational", "entityframework" } };
```

Comment:
This is a regression. Both `Microsoft.EntityFrameworkCore` and `Microsoft.EntityFrameworkCore.Relational` have the tags `entity` and `framework`, hence why they are top results. We will need to boost direct dependencies to remove the relational package from top results.